src/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.cpp

```diff
@@ -101,6 +105,108 @@ Mass Segment::Solution::computeDeltaMass() const
     return Mass::Kilograms(getInitialMass().inKilograms() - getFinalMass().inKilograms());
 }
 
+MatrixXd Segment::Solution::getDynamicsContribution(
```

Comment:
Nit: I think the order of the new 3 functions in the cpp is not the same as in the hpp.

Reply:
Thanks 🙏 Sorted